tests/conftest.py

```diff
@@ -66,3 +67,125 @@ def run_migrations() -> Iterator[None]:
             connection.execute(f"TRUNCATE TABLE IF EXISTS {database}.{table_name}")
 
     redis_client.flushdb()
+
+
+@pytest.fixture
+def convert_legacy_to_snql() -> Iterator[Callable[[str, str], str]]:
+    def convert(data: str, entity: str) -> str:
+        legacy = json.loads(data)
+
+        def func(value: Union[str, List[Any]]) -> str:
+            if not isinstance(value, list):
+                return value
+
+            children = ",".join(map(func, value[1]))
+            alias = f" AS {value[2]}" if len(value) > 2 else ""
+            return f"{value[0]}({children}){alias}"
+
+        def literal(value: Union[str, List[Any]]) -> str:
+            if isinstance(value, list):
+                return f"tuple({','.join(list(map(literal, value)))})"
```

Comment:
I feel we will soon want to provide a simpler syntax for tuples. Probably just  `(a, b, c)`
No need to do it at this point, but we may want to keep it in mind.

Reply:
I actually started to add that, but then I realized that when we parse the old syntax we convert array literals to `tuple(...)` in the AST. So it didn't seem worth adding a bunch of complicated logic. It might be worth revisiting this in the future.